app.js

```diff
@@ -49,6 +50,31 @@ setInterval(() => {
   });
 }, 60 * 60 * 1000);
 
+async function sendReminderEmail() {
+  const users = await db.users.findAll({
+    where: { updated_at: { [Op.lt]: moment().subtract(7, 'days').toDate() } },
```

Comment:
This would send out emails to unconfirmed users as well as users who already created their account